opm/simulators/wells/WellState.cpp

```diff
@@ -569,7 +569,8 @@ void WellState::reportConnections(std::vector<data::Connection>& connections,
     }
     for( auto& comp : connections) {
         const auto * rates = &perf_data.phase_rates[np*local_comp_index];
```

Comment:
It does looks like it is related to completion here, although we use `local_comp_index`.